pupil_src/shared_modules/marker_tracker_3d/optimization/model_state.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import os
+
+import numpy as np
+
+import file_methods
+from marker_tracker_3d import utils
+
+logger = logging.getLogger(__name__)
+
+
+class ModelState:
+    def __init__(self, save_path):
+        self.save_path = save_path
+
+        self.current_frame_id = 0
+
+        self.frames_id = []
```

Comment:
This is a minor remark, but I also saw it at other places, so I thought we might discuss it. Is there a specific reason why this is `frames_id` and not `frame_ids`? The latter would sound better to me, because I feel the list contains multiple ids which are each linked to a single frame.